easybuild/easyconfigs/f/FHI-aims/FHI-aims-240920-intel-2024a.eb

```diff
@@ -0,0 +1,60 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+# Updated to 240507
+# Help from the FHI-aims developers and Alexander Grund to move to oneAPI is greatly appreciated
+# J. Sassmannshausen (Imperial College London/UK)
+##
+
+easyblock = 'CMakeMake'
+
+name = 'FHI-aims'
+version = '240920'
+
+homepage = 'https://fhi-aims.org/'
+description = """FHI-aims is an efficient, accurate all-electron,
+full-potential electronic structure code package for computational molecular
+and materials science (non-periodic and periodic systems). The code supports
+DFT (semilocal and hybrid) and many-body perturbation theory. FHI-aims is
+particularly efficient for molecular systems and nanostructures, while
+maintaining high numerical accuracy for all production tasks. Production
+calculations handle up to several thousand atoms and can efficiently use (ten)thousands of cores.
+"""
+
+toolchain = {'name': 'intel', 'version': '2024a'}
+toolchainopts = {'opt': True, 'precise': True, 'usempi': True, 'oneapi': True}
+
+download_instructions = """
+The source code must be downloaded manually from the FHI-aims club
+(https://fhi-aims.org/get-the-code-menu/login).
+Access to the FHI-aims club requires a valid license and registration.
+Details on available license options and how to register to access
+FHI-aims club may be found at:
+https://fhi-aims.org/get-the-code-menu/get-the-code """
+
+sources = ['%(namelower)s.%(version)s.tgz']
+checksums = ['f26d6bede3f5c6be73403f46f83a8b082964f09d5ee875b6ae53decad96eeccd']
+
+builddependencies = [('CMake', '3.29.3')]
+
+configopts = ' -DCMAKE_Fortran_COMPILER="$MPIF90" -DCMAKE_CXX_COMPILER="$MPICXX"  -DCMAKE_C_COMPILER="$MPICC" '
+configopts += ' -DLIBS="mkl_scalapack_lp64 mkl_blacs_intelmpi_lp64 mkl_intel_lp64 mkl_sequential mkl_core" '
```

Comment:
If it can take $LIBSCALAPACK then that would be better.

Reply:
Good idea. So I tried:
```
-DLIBS="$LIBSCALAPACK"
```
and it fails with:
```
CMake Error at cmake/modules/Tools.cmake:87 (message):
  Could not find -Wl,-Bstatic
```
As:
```
echo $LIBSCALAPACK 
-Wl,-Bstatic -Wl,--start-group -lmkl_scalapack_lp64 -lmkl_blacs_intelmpi_lp64 -lmkl_intel_lp64 -lmkl_sequential -lmkl_core -Wl,--end-group -Wl,-Bdynamic
```